connector_ecommerce/sale.py

```diff
@@ -291,12 +291,10 @@ def action_view_parent(self, cr, uid, ids, context=None):
         action['res_id'] = parent.id
         return action
 
-    # TODO: remove in odoo 8.0
+    # XXX the 3 next methods seems very specific to magento
```

Comment:
Is there a special reason for removing the `TODO: remove in odoo 8.0`?
The methods are really to remove (I will do it in another PR I think)